src/gui/clips/PatternClipView.cpp

```diff
@@ -155,6 +159,11 @@ void PatternClipView::paintEvent(QPaintEvent*)
 			embed::getIconPixmap( "muted", size, size ) );
 	}
 	
+	if (m_marker)
+	{
+		p.drawLine(m_markerPos, rect().bottom(), m_markerPos, rect().top());
+	}
```

Comment:
And here too. `SampleClipView` doesn't have this.

Reply:
`SampleClipView` does have this, but it was already implemented before this PR, so it doesn't show up on the diff.